### ISSUE TITLE
Uncaught node.js Error

### ISSUE THREAD

**Issue by bjornfor** [matejc/nixui#15]
I tried nixui-0.1.1 from nixpkgs master branch. It still crashes with an "Uncaught node.js Error", although the traceback is different now. Now it complains

```
...
while evaluating the attribute ‘"${attr}"’ at "/etc/nixos/configuration.nix":803:7:
attribute ‘networking.hostName’ missing, at "/etc/nixos/configuration.nix":804:12

nix-instantiate exited with status: 1
```

My config (obviously) work with nixos-rebuild, so I don't know what's going on here.

If you want to debug this, here is my config:

https://github.com/bjornfor/nixos-config/blob/master/configuration.nix

**Comment by matejc** [comment]
Thanks for reporting this issue, I changed a bit the way of reading the configuration file, I have reproduced the error with your configuration file and now it works. The fix will be in version 0.1.2.

**Comment by bjornfor** [comment]
Thanks.
